### PR TITLE
Remove openSUSE Leap 15.4 due to EOL

### PR DESCRIPTION
**Pull request by nodeg**
## What does this PR change?

openSUSE 15.4 is EOL since the end of 2023. See https://en.opensuse.org/Lifetime
I did not touch the cloud files, though.

### CODE REVIEW

File: main.tf.libvirt-testsuite.example [hunk 1]
```diff
@@ -34,7 +34,7 @@ module "cucumber_testsuite" {
   # "amazonlinux2o",
   # "centos7o", "centos8o", "centos9o",
   # "libertylinux9o",
-  # "opensuse154o", "opensuse155o",
+  # "opensuse156o", "opensuse155o",
```
Comment (Bischoff) [inline main.tf.libvirt-testsuite.example:37]: sort by increasing number please, like we do for centos, for consistency

File: main.tf.libvirt-testsuite.example.Manager-43 [hunk 2]
```diff
@@ -34,14 +34,14 @@ module "cucumber_testsuite" {
   # "amazonlinux2o",
   # "centos7o", "centos8o", "centos9o",
   # "libertylinux9o",
-  # "opensuse154o", "opensuse155o",
+  # "opensuse156o", "opensuse155o",
```
Comment (Bischoff) [inline main.tf.libvirt-testsuite.example.Manager-43:37]: same

File: main.tf.libvirt-testsuite.example (hunk 1 above)
Comment (vandabarata) [inline main.tf.libvirt-testsuite.example:37]: nitpicky, I know, but... 
```suggestion
  # "opensuse155o", "opensuse156o",
```

File: main.tf.libvirt-testsuite.example.Manager-43 (hunk 2 above)
Comment (vandabarata) [inline main.tf.libvirt-testsuite.example.Manager-43:37]: same here :sweat_smile:

**Review by Bischoff — CHANGES_REQUESTED** [review]
see nitpicks

File: main.tf.libvirt-testsuite.example (hunk 1 above)
Comment (vandabarata) [reply main.tf.libvirt-testsuite.example:37]: we have the same nitpicks :rofl:

**Review by vandabarata — APPROVED** [review]

**Review by Bischoff — APPROVED** [review]

**Review by srbarrios — CHANGES_REQUESTED** [review]
Please fix the code in Sumaform tests before merging it.

**Comment by Bischoff** [comment]
> Please fix the code in Sumaform tests before merging it.

Sorry, i need those changes for my 4.3.14 BV and I don't have the time to look at that. It's also very likely to be a false positive. Merging.

**Review by Bischoff — CHANGES_REQUESTED** [review]
Please close. I fixed the PR in https://github.com/uyuni-project/sumaform/pull/1697

**Comment by Bischoff** [comment]
I fixed the PR in #1697

**Pull request closed without merging**